api/zulip/__init__.py

```diff
@@ -148,6 +156,7 @@ def generate_option_group(parser, prefix=''):
     return group
 
 def init_from_options(options, client=None):
+    # type: (Any, Any) -> Any 
```

Comment:
It won´t allow me to change por OptionParser nor client without an error. 
api/zulip/__init__.py:159: error: "OptionParser" has no attribute "zulip_client", 

Also the error when I tried with Client is this one:
api/zulip/__init__.py:162: error: Incompatible types in assignment (expression has type "str", variable has type "Client")

Are you sure that is the correct way? Also, shouldn´t the returned argument be a Client?